master/internal/core.go

```diff
@@ -921,11 +920,8 @@ func (m *Master) Run(ctx context.Context) error {
 	user.InitService(m.db, m.system, &m.config.InternalConfig.ExternalSessions)
 	userService := user.GetService()
 
-	m.proxy, _ = m.system.ActorOf(actor.Addr("proxy"), &proxy.Proxy{
-		HTTPAuth: processProxyAuthentication,
-	})
-
 	allocationmap.InitAllocationMap()
+	proxy.InitProxy(processProxyAuthentication)
```

Comment:
anything stopping us from moving auth to the proxy package?

Reply:
I'm not sure why it's defined in `api_auth` anyway as it's only used in relation to proxy -- will move it over in the next commit

Reply:
actually -- moving this over to proxy would cause an import cycle.